platform/android/export/export_plugin.cpp

```diff
@@ -1005,6 +1005,61 @@ void EditorExportPlatformAndroid::_write_tmp_manifest(const Ref<EditorExportPres
 	store_string_at_path(manifest_path, manifest_text);
 }
 
+void EditorExportPlatformAndroid::_fix_themes_xml(const Ref<EditorExportPreset> &p_preset) {
+	const String themes_xml_path = ExportTemplateManager::get_android_build_directory(p_preset).path_join("res/values/themes.xml");
+	bool enable_swipe_to_dismiss = p_preset->get("wear_os/swipe_to_dismiss");
+
+	if (!FileAccess::exists(themes_xml_path)) {
+		print_error("res/values/themes.xml does not exist.");
+		return;
+	}
+
+	String xml_content = "";
+	Ref<FileAccess> file = FileAccess::open(themes_xml_path, FileAccess::READ);
+	PackedStringArray lines = file->get_as_text().split("\n");
+	file->close();
+
+	// Check if the line contains the existing <item name="android:windowSwipeToDismiss"> element.
+	// If found and `enable_swipe_to_dismiss` is true, remove the item.
+	// If found and `enable_swipe_to_dismiss` is false, update its value to "false".
+	bool found = false;
+	bool modified = false;
+	for (int i = 0; i < lines.size(); i++) {
+		String line = lines[i];
+		if (line.find("<item name=\"android:windowSwipeToDismiss\">") != -1) {
+			if (enable_swipe_to_dismiss) {
+				lines.remove_at(i);
```

Comment:
Why removing it instead of setting its value to `true`?

Reply:
Aaah! No specific reasons—it can simply be set to true. I just thought removing it might be better.
@m4gr3d Let me know if you'd prefer me to change it!

Reply:
Let's set it to `true`, it makes the logic simpler to follow!
Let's also update the `themes.xml` file so the original version contains the `android:windowSwipeToDismiss` element. To your point, a user may still remove that element manually, so let's keep the logic below which adds it back if it's missing.

Reply:
@m4gr3d All done! Take a look.